assets/dev/js/editor/elements/views/base.js

```diff
@@ -661,10 +661,10 @@ BaseElementView = BaseContainer.extend( {
 			dataBinding.el.getAttribute( 'data-binding-setting' ) === changedControl;
 	},
 
-	async getDynamicValue( settings, bindingSetting ) {
```

Comment:
I originally made changes in this function to address the task's AC in it, so in order for it to cover cases from previous logic and new logic, I had to make changes.
Since then, I decided to move my logic out because they don't rely on any dynamic content, so no need to wait for the async logic to happen to render them so these changes can be removed, however, I think we should consider leaving them if they make sense universally (regarding this function and the `renderDataBinding`, not to be confused with `renderDataBindings` )